### PR TITLE
Revert "extensions: Enable SCOS extension builds with COSA"

### PR DESCRIPTION
**Pull request by LorbusChris**
This reverts commit 7bcca86eb807f27e4b0899f7c4443ae9c0d153d2.

When running `cosa build-extensions-container`, the `src/config.json` file is not present, so the condition for setting a non-empty variant will never be true.

This can be mitigated by running `cosa update-variant default scos` beforehand, pointing the symlinks for `{manifest,extensions}.yaml` to the desired variant.

/cc @jmarrero 
/cc @travier 

I think it might be best to have COSA pass the variant to the Dockerfile as an arg, but for now this can be worked around per the above.

### CODE REVIEW

**Comment by jmarrero** [comment]
I see, so we don't need to change anything for this release? SCOS pipeline will use the correct repos by calling `cosa update-variant default scos` ?

**Review by jmarrero — APPROVED** [review]
lgtm

**Comment by cgwalters** [comment]
> When running cosa build-extensions-container, 

That's one possible path for building the extensions container.  But, note that part of our design for the extensions container is that it has a `Dockerfile` that can also be built using any container build system.  (`cosa build-extensions-container` is just a wrapper for that)

Unless OKD has an explicit reason to use cosa for this, I'd suggest using a container-native build process for it.  We already have one in Prow:
https://github.com/openshift/release/blob/ceac95613d29f9c51b397757f6272f03a1932f86/ci-operator/config/openshift/os/openshift-os-master__periodic.yaml#L102

**Comment by LorbusChris** [comment]
Should we get this in?

**Comment by jmarrero** [comment]
I guess I need to standardize my lgtm to /lgtm if not I will keep missing these.

**Comment by travier** [comment]
Note that `cosa update-variant` is not meant to be used in CI as it changes the content of the repo. It's only there for version updates.

**Pull request merged**
